### DISCUSSION TITLE
Usability Issue - Discussion Filtering - Options reflecting interest/following vs favourites

### DISCUSSION THREAD

**Discussion by ericmarceau-rogers** [aclap-dev/video-downloadhelper#18] [Help and Q&A] [answered]
### Your question

Is there a mechanism that can be implemented to flag a discussion as being "**of interest**" or to be "**followed**"?  

And that the flag be distinct from another flag to indicate "**favourite**"?

Is there some way that you can add a "**standard**" filter (_drop-down ?_) such that

- the Discussion window will **only show** those items to which we **personally responded to**?
- the Discussion window will **only show** those items which we flagged as "of interest" or "following"?

Thank you for your consideration,

Eric
_just a user_

### Addon & CoApp version and details

Any / All

**Comment by GrampaWildWilly** [comment]
When you click your account avatar in the upper right corner of any GitHub page, it drops down a menu.  On that menu, towards the bottom, there is a link to GitHub Docs.  If your answer is anywhere, it's in there.

**Comment by paulrouget** [comment, marked as answer]
> Is there a mechanism that can be implemented to flag a discussion as being "of interest" or to be "followed"?

There's a subscribe button on the right of a discussion.

> And that the flag be distinct from another flag to indicate "favourite"?

I don't believe so.

> Is there some way that you can add a "standard" filter (drop-down ?) such that
> the Discussion window will only show those items to which we personally responded to?
> the Discussion window will only show those items which we flagged as "of interest" or "following"?

The dropdown is a github function, not something we can change.
